examples/notebooks/beam-ml/run_inference_tensorflow.ipynb

```diff
@@ -0,0 +1,489 @@
+        "with tf.io.TFRecordWriter(predict_values_five_times_table) as writer:\n",
+        "  for i in value_to_predict:\n",
+        "    example = ExampleProcessor().create_example(feature=i)\n",
+        "    writer.write(example.SerializeToString())"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Fit The Model\n",
+        "\n",
+        "This example builds a model. Since RunInference requires pretrained models, this segment builds a usable model."
+      ],
+      "metadata": {
+        "id": "G-sAu3cf31f3"
+      }
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "AnbrxXPKeAOQ",
+        "outputId": "0574f487-5624-4968-e6f5-110e79c40210"
+      },
+      "source": [
+        "RAW_DATA_TRAIN_SPEC = {\n",
+        "'x': tf.io.FixedLenFeature([], tf.float32),\n",
+        "'y': tf.io.FixedLenFeature([], tf.float32)\n",
+        "}\n",
+        "\n",
+        "dataset = tf.data.TFRecordDataset(example_five_times_table)\n",
+        "dataset = dataset.map(lambda e : tf.io.parse_example(e, RAW_DATA_TRAIN_SPEC))\n",
+        "dataset = dataset.map(lambda t : (t['x'], t['y']))\n",
+        "dataset = dataset.batch(100)\n",
+        "dataset = dataset.repeat()\n",
+        "\n",
+        "model.fit(dataset, epochs=500, steps_per_epoch=1, verbose=0)"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "<keras.callbacks.History at 0x7fc0a24b8090>"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 40
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Save the Model"
+      ],
+      "metadata": {
+        "id": "r4dpR6dQ4JwX"
+      }
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "fYvrIYO3qiJx"
+      },
+      "source": [
+        "RAW_DATA_PREDICT_SPEC = {\n",
+        "'x': tf.io.FixedLenFeature([], tf.float32),\n",
+        "}\n",
+        "\n",
+        "@tf.function(input_signature=[tf.TensorSpec(shape=[None], dtype=tf.string , name='examples')])\n",
+        "def serve_tf_examples_fn(serialized_tf_examples):\n",
+        "  \"\"\"Returns the output to be used in the serving signature.\"\"\"\n",
+        "  features = tf.io.parse_example(serialized_tf_examples, RAW_DATA_PREDICT_SPEC)\n",
+        "  return model(features, training=False)\n",
+        "\n",
+        "signature = {'serving_default': serve_tf_examples_fn}\n",
+        "\n",
+        "tf.keras.models.save_model(model, save_model_dir_multiply, signatures=signature)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Run the Pipeline\n",
```

Comment:
Could we add a keyed example after this? Could just be a single section similar to what we do in sklearn

Reply:
I added a keyed example.  I think it's nice as it also illustrates another maybe more simple way to construct TF protos.